src/swap/calculateEthForNxm.ts

```diff
@@ -1,19 +1,19 @@
 import { Reserves } from './reserves.type';
 
+/**
+ * Calculates the amount of eth flowing out of the pool for the given amount of nxm flowing in
+ */
+// suggestion: calculateEthOutGivenNxmIn
 export const calculateEthForNxm = (nxmIn: bigint, reserves: Reserves): bigint => {
```

Comment:
What do you think about renaming this function to the suggestion above, or something else more verbose? All four functions have a suggestion above it